heron/statemgrs/src/python/statemanager.py

```diff
@@ -122,7 +122,7 @@ def establish_ssh_tunnel(self):
     return localport
 
   def terminate_ssh_tunnel(self):
-    if self.tunnel:
+    if hasattr(self, 'tunnel'):
```

Comment:
What if it has the attribute but the value is None? Should we be doing this?
```
if hasattr(self, 'tunnel') and self.tunnel:
```
or is that redundant? Not sure if `hasattr` also covers `self.tunnel not None`